spec/bolt/cli_spec.rb

```diff
@@ -161,6 +161,33 @@ def stub_config(file_content = {})
             }.to raise_error(Bolt::CLIExit)
           }.to output(/Available actions are:.*upload/m).to_stdout
         end
+
```

Comment:
We should also add a test that `bolt help subcommand action` help text doesn't include invalid options or usage text. I think `plan show` and `plan run` would be good, to ensure that options and help text are present when expected and not present when not expected.